src/main/java/seedu/dietmanager/logic/commands/recipe/BuildNewRecipeCommand.java

```diff
@@ -89,8 +89,8 @@ public Result getResult(Profile profile) {
                     + "Second parameter is activity level, choose from -- low/moderate/high.";
         } else {
             if (maxNumOverflow) {
-                this.resultString = "We support at most 4 kinds of food in a meal, "
-                        + "otherwise it's not good for your health!\n\n";
+                this.resultString = "We support at most 3 kinds of food in a meal, "
+                        + "otherwise it's easy to overtake calories and not good for your health!\n\n";
```

Comment:
What's with this recommendation? Since we are going to fill up on the recommended caloric intake anyway will this lead to too much food being left for the night?